docs/src/configuring-zed.md

```diff
@@ -29,10 +29,26 @@ Extensions that provide language servers may also provide default settings for t
 
 # Settings
 
-## Active Pane Magnification
+## Active Pane Modifiers
+
+Styling settings applied to the active pane.
+
+### Magnification
 
 - Description: Scale by which to zoom the active pane. When set to `1.0`, the active pane has the same size as others, but when set to a larger value, the active pane takes up more space.
-- Setting: `active_pane_magnification`
+- Setting: `magnification`
+- Default: `1.0`
+
+### Border size
+
+- Description: Size of the border surrounding the active pane. When set to 0, the active pane doesn't have any border. The border is drawn inset.
+- Setting: `border_size`
+- Default: `0.0`
+
+### Inactive Opacity
+
+- Description: /// Opacity of inactive panels. When set to 1.0, the inactive panes have the same opacity as the active one. If set to 0, the inactive panes content will not be visible at all.
```

Comment:
> ///

Let's remove that and mention ranges around, where it matters.

Reply:
oops, fixed, and the range is implicit by mentioning both extreme values no? or would you like something more specific? any suggestion for how to word this better?

note: the only value with ranges is the opacity, the others don't have limits that's why I didn't mention anything.

Reply:
More like [0.0, 1.0] range that's not anywhere here and in the `///` docs too, just in the json defaults?

What's up with negatives, btw, are those clamped to 0.0?

Reply:
alright, added explicit mention of the range

> What's up with negatives, btw, are those clamped to 0.0?

yep, as far as I understand from the `clamp` docs, any value `<0` will be clamped to `0.0`